### ISSUE TITLE
Create a UI to copy Maxmind DB's or IPinfo DB's into Graylog Clusters

### ISSUE THREAD

**Issue by jivepig** [Graylog2/graylog2-server#11380]
Right now, it is a manual process to copy all the geoip databases into the nodes that require it.
- add a way via a new or existing UI to install/push them so they install to all nodes in the cluster.

## Why?
Saves time,
very accurate and takes steps out of permissions issues on the files and the naming of them.

## Your Environment
4.1 

* Graylog Version: 4.1
* Elasticsearch Version: n/a
* MongoDB Version: n/a
* Operating System:n/a
* Browser version: n/a

**Comment by jivepig** [comment]
@bud1979
@danotorrey

**Comment by danotorrey** [comment]
Good point @boosty. This was also discussed in the context of Graylog Cloud, so I wanted to link that conversation here too for review. I think this comment has the latest results from discussion on the topic earlier this year: 
https://github.com/Graylog2/graylog-plugin-cloud/issues/535#issuecomment-739960302

**Comment by boosty** [comment]
Thanks @danotorrey. For Graylog Cloud the idea is that we provide the file by default. So this feature is mostly interesting for non-cloud cases.

**Comment by jivepig** [comment]
This adds more context with Docker. It gets difficult to map geoip databases.

**Comment by jivepig** [comment]
issue resolved adding S3 buckets or external storage for location.

**Comment by boosty** [comment]
@jivepig Ok if we close this?

As you mentioned, Graylog allows downloading the files from Cloud storage by now, as documented on https://go2docs.graylog.org/current/making_sense_of_your_log_data/geolocation.htm.

A real simplification for the users could be https://github.com/Graylog2/graylog2-server/issues/22361, which I would suggest to keep open.

**Comment by jivepig** [comment]
> [@jivepig](https://github.com/jivepig) Ok if we close this?
> 
> As you mentioned, Graylog allows downloading the files from Cloud storage by now, as documented on https://go2docs.graylog.org/current/making_sense_of_your_log_data/geolocation.htm.
> 
> A real simplification for the users could be [#22361](https://github.com/Graylog2/graylog2-server/issues/22361), which I would suggest to keep open.

Any possible way we can automate this process for customers is a good idea.
